### PR TITLE
Handle the new hint ShowShortcutsInContextMenus

### PR DESCRIPTION
**Pull request by tsujan**
Fixes https://github.com/lxde/lxqt/issues/1426.

ShowShortcutsInContextMenus is false by default and, contrary to it name, it isn't just about context menus.

### CODE REVIEW

File: src/lxqtplatformtheme.cpp [hunk 1]
```diff
@@ -302,7 +302,10 @@ QVariant LXQtPlatformTheme::themeHint(ThemeHint hint) const {
         break;
     case WheelScrollLines:
         return wheelScrollLines_;
-        break;
```
Comment (tsimonq2) [inline src/lxqtplatformtheme.cpp]: Why is this line removed?

Comment (tsujan) [reply src/lxqtplatformtheme.cpp]: was redundant

**Comment by towhee** [comment]
Why can't you this optional?  Modern ui are moving away from the classic main menubars.

**Comment by tsujan** [comment]
> Modern ui are moving away from the classic main menubars.

"Modern"?! We stick to the classic.

**Comment by towhee** [comment]
Seriously I'm trying to understand Qt's thinking here.  Qt goes to great effort to allow users to customize the app appearance with styles and style sheets.  Why limit the user?  I understand your change as the default behavior - I just want it to be optional.  

How about adding setShowShortcutsInContextMenus(bool)

Pretty please :*)

**Comment by tsujan** [comment]
I, for one, never have problem with more customization -- and my PRs prove that --  but *this* patch isn't about customizability. Qt devs changed a default behavior in a backward incompatible way and without caring about usibility (I guess they didn't think about the side effects of their decision for Linux). This patch just reversed that.

If removal of shortcuts in context menus *by default* is called "modern", then "modern" isn't a good thing.
